content/posts/virtualbox.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Setup Virtualbox"
+date: 2021-03-19
+draft: false
+description: "Oracle Virtualmachine & ansible"
+tags: ["Virtualbox", "ansible", "how to", "installation", "oracle" , "Windows", "Virtualization"]
+author: "ShowMeYourSkil"
+---
+
+![image](https://user-images.githubusercontent.com/39274150/111812404-64c6dd80-88d8-11eb-9492-81d771594c16.png)
+
+# Virtualbox
+
+Hi and welcome back to my new article, 
+In this article I would like to tell you about the virtualization software of Oracle Virtualbox. 
+I will deal with the issues, what it can do and how to install it.
+
+# Why I use the Oracle Virtualbox?
+
+I use the virtualbox to easily install virtual machines running Linux on my Windows PC. 
+When I no longer need them I can simply delete them again. 
+Oracle Virtualbox is the most popular and in my opinion the best virtualization software for Windows, Mac OS, Linux and Oracle Solaris. 
+The virtual machine offers a large number of operating systems.
+Currently I use the virtualbox for testing with the automation platform ansible. 
+If you want to know how to install ansible please read [this](https://mt32.net/posts/ansible-install/) article.
+
+# How to Install Oracle Virtualbox on Windows10.
+
+Example OS: Windows 10
+
+1. Open: `https://www.virtualbox.org/`
+2. Press the big button: `Download Virtualbox 6.1`
+3. Download the setup.exe file
+4. Execute the Setup file
+5. Follow the setup instructions
+
+# How to install a Virtualmachine
+
+1. Open the Virtualbox
+2. Click the button `new` to create a new machine
+3. Now you must set a Name, a folder, the type and the version
+    - In my case i use the type `Linux`
+    - For the version can you now set the Linux system. In my case I use `Ubuntu 64bit` 
+4. Now you must set the Memory size
+5. Congratulations! Now you have your own virtual machine.
+
+# How to start a Virtualmachine
```

Comment:
Where is the ISO part?

Reply:
Step 3.2 

`3. Now you must set a Name, a folder, the type and the version
    - In my case i use the type `Linux`
    - For the version can you now set the Linux system. In my case I use `Ubuntu 64bit``